src/aggregator/aggregator/aggregator.go

```diff
@@ -809,7 +813,10 @@ func (m *aggregatorAddMetricErrorMetrics) ReportError(err error) {
 		m.valueRateLimitExceeded.Inc(1)
 	case xerrors.Is(err, errArrivedTooLate):
 		m.arrivedTooLate.Inc(1)
+	case xerrors.Is(err, errAggregationClosed):
+		m.aggregationClosed.Inc(1)
 	default:
+		log.Error("uncategorized error", zap.Error(err))
```

Comment:
Worth just always logging? It may get spammy but that's probably fine in error scenarios

Reply:
we already have logs for some (maybe all?) the actual categorized errors, like too late. maybe a clean up would be to log in a central location, but that's more work than I want to do :)

i'm not worried about the spam, since we have a sampler config for zap